crates/configuration/src/shared/node.rs

```diff
@@ -263,8 +266,8 @@ impl Default for NodeConfigBuilder<Initial> {
                 image: None,
                 command: None,
                 args: vec![],
-                is_validator: false,
-                is_invulnerable: false,
+                is_validator: true,
+                is_invulnerable: true,
```

Comment:
All nodes are validator and invulnerable by default ?

Reply:
Yes, that the current behavior. We want to follow the same logic. If you need `full_nodes` you need to make it explicit.